bash/jhail

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source "$(dirname $0)"/setup_env
+
+jupyter nbextension list 2>&1 | grep 'jupyter_spark *enabled'
+nb_extension_enabled=$?
+jupyter nbextension list 2>&1 | grep 'jupyter_spark *OK'
+nb_extension_validated=$?
+jupyter serverextension list 2>&1 | grep 'jupyter_spark *enabled'
+server_extension_enabled=$?
+jupyter serverextension list 2>&1 | grep 'jupyter_spark *OK'
```

Comment:
Also, this seems kinda slow, but maybe we don't mind compared to the startup time of a notebook:

```
$ time jupyter serverextension list
real	0m0.607s
user	0m0.469s
sys	0m0.123s
```